pkg/cmd/server/api/v1/conversions.go

```diff
@@ -62,6 +62,16 @@ func init() {
 				obj.ExecHandlerName = DockerExecHandlerNative
 			}
 		},
+		func(obj *ServingInfo) {
+			if len(obj.BindNetwork) == 0 {
+				obj.BindNetwork = "tcp4"
```

Comment:
defaulting here means this field will appear everywhere we use ServingInfo. This PR doesn't make use of it for kubelet or etcd... is that misleading?